docs/modules/kat.md

```diff
@@ -0,0 +1,8 @@
+---
+Name: KAT
+URL: https://github.com/TGAC/KAT
+Description: >
+    The K-mer Analysis Toolkit (KAT) contains a number of tools that analyse and compare K-mer spectra.
+---
+
+The KAT multiqc module interprets output from KAT distribution analysis json files.
```

Comment:
A fraction more description, eg. what the tool is typically used for, what people can look for in the results etc is welcome..